acj/api/ltilaunch.py

```diff
@@ -91,6 +98,48 @@ def status():
         "course_role": String #the SystemRole string value for the current_user
     }
 }
-    todo: query db for data to return, assemble into JSON object, return obj
     """
-    return "some JSON"#jsonify({"error": 'Invalid login data format. Expecting json.'}), 400
+    isLTISession = False
+    if sess.get('LTI'):
+        isLTISession = True
+    
+    body = {}
+    body['valid'] = isLTISession
+
+    assignment = {}
+    assignment['id'] = ""
+    assignment['exists'] = False
+    lti_resource_link = LTIResourceLink.query.get(sess['lti_resource_link'])
+    if lti_resource_link:
+        assignment['id'] = lti_resource_link.acj_assignment_id
+        if assignment['id']:
+            assignment['exists'] = True
+
+    course = {}
+    lti_context = LTIContext.query.get(sess['lti_context'])
+    course['id'] = ""
+    course['exists'] = False
+    if lti_context:
+        course['id'] = lti_context.acj_course_id
+        if course['id']:
+            course['exists'] = True
+
+    user = {}
+    user['exists'] = False
+    lti_user = LTIUser.query.get(sess['lti_user'])
+    if lti_user:
+        if lti_user.user_oauth_id:
+            user['exists'] = True
+
+    user['course_role'] = ""
+    lti_user_resource_link = LTIUserResourceLink.query.get(sess['lti_user_resource_link'])
+    if lti_user_resource_link:
+        if lti_user_resource_link.course_role:
+            roleSubstring = str(lti_user_resource_link.course_role).split(".")[1]
+            user['course_role'] = roleSubstring
```

Comment:
you can rewrite this to `user['course_role'] = lti_user_resource_link.course_role.value` to get the string value out of Enum more easily